### PR TITLE
Replace verbosity flags by base-logging macros

### PR DESCRIPTION
**Pull request by hemker**
@2maz Fix missing check for verbosity

### CODE REVIEW

**Comment by hemker** [comment]
@2maz I replaced all verbosity/std::cout occurences by base-logging macros, so the package can be controlled via log level variables.

File: CMakeLists.txt [hunk 1]
```diff
@@ -5,3 +5,5 @@ find_package(Rock)
 rock_activate_cxx11()
 rock_init(pocolog_cpp 0.1)
 rock_standard_layout()
+
+add_compile_options(-DBASE_LOG_NAMESPACE=${PROJECT_NAME})
```
Comment (2maz) [inline CMakeLists.txt:9]: This is already part of [rock_init](https://github.com/rock-core/base-cmake/blob/7c60186292d3a18794ef6f7fb7d23e1f72ed5d56/modules/Rock.cmake#L130)

File: src/FileStream.cpp [hunk 2]
```diff
@@ -55,14 +56,14 @@ bool pocolog_cpp::FileStream::open(const char* fileName, std::ios_base::openmode
     writePos = 0;
     this->fileName = fileName;
     
-//     std::cout << "File opened " << (fd > 0) << " file size " << fileSize  << " blk size " << blockSize << std::endl;
+    LOG_DEBUG_S << "File opened " << (fd > 0) << " file size " << fileSize  << " blk size " << blockSize << std::endl;
```
Comment (2maz) [inline src/FileStream.cpp:59]: LOG_DEBUG_S comes with an std::endl at the end, so you could remove it here (holds for the other statements as well)

**Review by 2maz — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (hemker) [reply CMakeLists.txt]: Done

File: src/FileStream.cpp (hunk 2 above)
Comment (hemker) [reply src/FileStream.cpp:59]: I see, there are a lot of std::endl left in the code. I'm going to remove them.

Comment (hemker) [reply src/FileStream.cpp]: Done

**Review by 2maz — APPROVED** [review]

**Pull request merged**